package.json

```diff
@@ -40,16 +40,16 @@
     "babel-polyfill": "^6.23.0",
     "lodash": "^4.12.0",
     "paths-js": "^0.4.5",
-    "react-native-svg": "~4.5.0"
+    "react-native-svg": "~5.1.8"
   },
   "devDependencies": {
     "babel-jest": "*",
     "babel-preset-react-native": "^1.9.0",
     "diff": "^3.1.0",
     "jest": "^18.0.0",
     "jest-react-native": "*",
-    "react": "~15.4.1",
-    "react-native": "~0.41.0",
+    "react": "16.0.0-alpha.6",
```

Comment:
No, believe it or not depending on this "alpha" version is actually the "correct" thing to do. Many people like you and I have this same reaction and the official answer seems to be "this is expected":

https://github.com/facebook/react-native/issues/13291
https://github.com/facebook/react-native/issues/13469

Also, we are depending on react-native-svg 5.1.8 which has this to say on [its readme](https://github.com/react-native-community/react-native-svg):

> react-native-svg >= 5.1.8 only supports react-native >= 0.44.0 and react == 16.0.0-alpha.6